Makefile

```
@@ -743,125 +704,51 @@ $(OCRD_EXECUTABLES:%=%-check):
 .PHONY: $(OCRD_EXECUTABLES:$(BIN)/%=%)
 $(OCRD_EXECUTABLES:$(BIN)/%=%): %: $(BIN)/%
 
-XDG_DATA_HOME ?= $(if $(HOME),$(HOME)/.local/share,/usr/local/share)
-DEFAULT_RESLOC ?= $(XDG_DATA_HOME)/ocrd-resources
+$(OCRD_EXECUTABLES): $(ACTIVATE_VENV)
 
 # do not delete intermediate targets:
 .SECONDARY:
 
 # suppress all built-in suffix rules:
 .SUFFIXES:
 
-# allow installing system dependencies for all modules
-# (mainly intended for docker, not recommended to use directly for live systems)
-# reset ownership of submodules to that of ocrd_all
-# (in case deps-ubuntu has been used with sudo and some modules were updated)
 # install git and parallel first (which is required for the module updates)
 deps-ubuntu:
 	apt-get -y install git parallel
```

```suggestion
	apt-get -y install git parallel jq
```

We need jq for getting the `.tools` keys from the `ocrd-tool.json`, right?

---

Yes. We should better do that with Python itself, right?